contributors/devel/sig-architecture/api_changes.md

```diff
@@ -1191,7 +1198,7 @@ provides some details
 
 In future Kubernetes versions:
 
-* if the feature progresses to beta or stable status, the feature gate can be removed or be enabled by default.
+* if the feature progresses to stable status, the feature gate can be removed or be enabled by default.
```

Comment:
drop this change, this section of the doc is not referring to a new beta API version/endpoint

Reply:
Droped.